### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Shivam250702**
Removed Punctuation marks and grammatical error

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

File: README.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ The Web UI can be found at http://localhost:9000 while the SMTP port will be on
 
 ### Support IPv6
 
-In case you are deploying to a cluster that uses only IPv6. You can use a custom command to pass a parameter to the Next.js start command
+In case you are deploying to a cluster that uses only IPv6,You can use a custom command to pass a parameter to the Next.js start command
```
Comment (catalinpit) [inline README.md:232]: "[...] uses only **IPv6,You** can use a custom [...]" should be "[...] uses only **IPv6, you** can use a custom [...]".

File: README.md [hunk 2]
```diff
@@ -203,13 +203,13 @@ We support DevContainers for VSCode. [Click here to get started.](https://vscode
...
 
-🚧 Docker containers and images are current in progress. We are actively working on bringing a simple docker build and publish pipeline for Documenso.
+🚧 Docker containers and images are currently in progress. We are actively working on bringing a simple docker build and publish pipeline for Documenso.
 
 ## Self Hosting
 
 We support a variety of deployment methods, and are actively working on adding more. Stay tuned for updates!
 
-> Please note the below deployment methods are for v0.9, we will update these to v1.0 once it has been released.
+> Please note the below deployment methods are for v0.9, we will update them to v1.0 once it has been released.
```
Comment (catalinpit) [inline README.md:212]: Just a suggestion, but maybe `are for v0.9, **and** we will update` will sound better.

File: README.md [hunk 3]
```diff
@@ -156,7 +156,7 @@ npm run d
 
 ### Manual Setup
 
-Follow these steps to setup documenso on you local machine:
+Follow these steps to setup documenso on your local machine:
```
Comment (catalinpit) [inline README.md:159]: Documenso should be capitalized, I'd say.

File: README.md [hunk 4]
```diff
@@ -70,10 +70,10 @@ Join us in creating the next generation of open trust infrastructure.
 
 ## Community and Next Steps 🎯
 
-We're currently working on a redesign of the application, including a revamp of the codebase so Documenso can be more intuitive to use and robust to develop upon.
+We're currently working on a redesign of the application, including a revamp of the codebase, so Documenso can be more intuitive to use and robust to develop upon.
 
 - Check out the first source code release in this repository and test it
-- Tell us what you think in the [Discussions](https://github.com/documenso/documenso/discussions)
+- Tell us what you think in the [Discussions](https://github.com/documenso/documenso/discussions).
```
Comment (catalinpit) [inline README.md:76]: Full stop not needed

File: README.md [hunk 5]
```diff
@@ -111,25 +111,25 @@ Contact us if you are interested in our Enterprise plan for large organizations
 
 ### Requirements
 
-To run Documenso locally you will need
+To run Documenso locally, you will need
```
Comment (catalinpit) [inline README.md:114]: Does `Running Documenso locally requires:` sound better? What do you think?

File: README.md [hunk 6]
```diff
@@ -70,10 +70,10 @@ Join us in creating the next generation of open trust infrastructure.
 
 ## Community and Next Steps 🎯
 
-We're currently working on a redesign of the application, including a revamp of the codebase so Documenso can be more intuitive to use and robust to develop upon.
+We're currently working on a redesign of the application, including a revamp of the codebase, so Documenso can be more intuitive to use and robust to develop upon.
```
Comment (catalinpit) [inline README.md:73]: Does `We're currently redesigning the application, including a revamp of the codebase, so Documenso can be more intuitive and robust to use and develop upon.` sound better? What do you think?

File: README.md [hunk 7]
```diff
@@ -111,25 +111,25 @@ Contact us if you are interested in our Enterprise plan for large organizations
...
+To run Documenso locally, you will need
 
 - Node.js
 - Postgres SQL Database
 - Docker (optional)
 
 ### Developer Quickstart
 
-> **Note**: This is a quickstart for developers. It assumes that you have both [docker](https://docs.docker.com/get-docker/) and [docker-compose](https://docs.docker.com/compose/) installed on your machine.
+> **Note**: This is a quickstart for developers. It assumes that you have both [docker](https://docs.docker.com/get-docker/), and [docker-compose](https://docs.docker.com/compose/) installed on your machine.
```
Comment (catalinpit) [inline README.md:122]: I don't think the comma is needed here

File: README.md (hunk 1 above)
Comment (catalinpit) [reply README.md:232]: There should also be a full stop at the end of the sentence.

**Review by catalinpit — CHANGES_REQUESTED** [review]
Thanks for your changes! Here are some more suggestions if you wish to contribute:

"4. Set the following environement variables." should be "4. Set the following environment variables."

---

"6. Run `npm run dev` root directory to start" should be "6. Run `npm run dev` **in the** root directory to start"

--- 

"To generate your own using these steps and a Linux Terminal or Windows Subsystem for Linux (WSL) see **[Create your own signing certificate](./SIGNING.md)**." should have a comma after `[...] Linux (WSL)`

---

No comma needed here "We support a variety of deployment methods`,` and are actively working on adding more."

---

Comma after the quickstart here "When using the developer quickstart an [Inbucket](https://inbucket.org/) server will be spun up in a docker container that will store all outgoing email locally for you to view."

Also `email` should be `emails`.

---

The sentence "The same can be done when using `npx` for one of bin scripts:" should contains `one of the bins`. The `the` is missing.

---

There should be a comma after `http://localhost:9000` in this sentence "The Web UI can be found at http://localhost:9000 while the SMTP port will be on localhost:2500."

---

Thanks once again!

File: README.md (hunk 7 above)
Comment (catalinpit) [reply README.md:122]: I checked with Grammarly, and it suggested removing the comma. 👀

**Comment by catalinpit** [comment]
> please try to merge that part which u think will improve your Doc/Readme

Hey @Shivam250702, sorry, I didn't get it. What should I merge?

**Pull request closed without merging**